REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia 0.6
-VideoIO 0.1.0
+julia 0.7
```

Comment:
Just to confirm: this is 0.7 because it is the _minimum_ requirement, this it works on 1.0 as well.

Reply:
yeah, that is the idea, right?

Reply:
yup you're good

Reply:
Great! You still need to merge though, I don't have write access to the upstream :P